datalad/config.py

```diff
@@ -129,7 +129,7 @@ def _get_default_file_candidates(self):
             cfg_file_candidates.append(opj(home_cfg_base_path, 'datalad.cfg'))
 
         # current dir config
-        cfg_file_candidates.append(opj('.datalad', 'config'))
+        cfg_file_candidates.append('datalad.cfg')
```

Comment:
ah right... since we are aiming for unification and don't think anyone would have those datalad.cfg laying around -- just comment it out entirely I guess. later it should read .datalad/config right?